tests/specs/scss/functions.scss

```diff
@@ -1,29 +1,73 @@
 
-@include test-module('Functions') {
+@include test-module("Functions") {
 
   // Testing set functions
-  @include test('config-set [function]') {
-    @include config-set('test.bar', 'foo');
-    @include config-set('test.color', #fff);
+  @include test("config-set [function] Set a single key.") {
+    @include config-set("bar", "foo");
+    @include config-set("color", #fff);
 
-    $expect: ("test": ("bar": 'foo', "color": #fff));
-    @include assert-equal($config-attr, $expect, 'Check if map is correct.');
+    $expect: ("bar": "foo", "color": #fff);
 
-    $config-attr: ();
+    @include assert-equal($config-attr, $expect, "Check if map is correct.");
+  }
+
+  // Testing set functions
+  @include test("config-set [function] Set a single !default key.") {
+    @include config-set("single-default", "foo", true);
+
+    $expect: ("single-default": "foo");
+    @include assert-equal($config-default, $expect, "Check if map is correct.");
+  }
+
+  // Testing set functions
+  @include test("config-set [function] Set a dotted key.") {
+    @include config-set("test.bar", "foo");
+    @include config-set("test.color", #fff);
+
+    $expect: ("bar": "foo", "color": #fff);
+    $config: config-get("test");
+
+    @include assert-equal($config, $expect, "Check if map is correct.");
   }
 
   // Testing get functions
-  @include test('config-get [function]') {
-    @include config-set('test.color', #000);
+  @include test("config-get [function] Get a value from key.") {
+    @include config-set("test.color", #000);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.